src/components/ClueSelector.js

```diff
@@ -67,7 +67,32 @@ export default function ClueSelector({ board }) {
 
   }, [clueColor]);
 
-  const clues = {red: redClues, blue: blueClues}[clueColor] || [];
+  const getClueString = (n, clues) =>{
+    let cluesByNum=clues.filter(( clue ) => clue.num == n);
```

Comment:
should use `===` here rather than `==`. Then you also need to make sure that they're both the same type, so you'd need to do `+clue.num`.